pkg/runtime/wfengine/wfengine.go

```diff
@@ -49,15 +47,32 @@ type WorkflowEngine struct {
 }
 
 var (
+	WorkflowActorType   = ""
+	ActivityActorType   = ""
```

Comment:
I don't feel great about having global variables that are mutable. Can we instead make these fields (or getter methods, like `func (wfe *WorkflowEngine) GetWorkflowActorType() string`) on the `WorkflowEngine` struct? I see that we initialize these in `NewWorkflowEngine` so I assume it would be easy to make this change. Or if we don't even need these at all, we can just remove them.

Reply:
Please comment if this approach looks fine, or you think something better can be done to propagate this info.